pkgs/applications/networking/syncthing/default.nix

```diff
@@ -34,6 +34,6 @@ stdenv.mkDerivation rec {
     description = "Open Source Continuous File Synchronization";
     license = stdenv.lib.licenses.mpl20;
     maintainers = with stdenv.lib.maintainers; [ pshendry joko peterhoeg ];
-    platforms = with stdenv.lib.platforms; linux ++ freebsd ++ openbsd ++ netbsd;
+    platforms = with stdenv.lib.platforms; linux ++ freebsd ++ openbsd ++ netbsd ++ darwin;
```

Comment:
There are binaries upstream for Solaris, so I'm guessing it would work on Illumos and thus fall under ```platforms.unix```. Nice catch.